manifest.json

```diff
@@ -23,10 +23,17 @@
     "service_worker": "src/routes/background/background.ts",
     "type": "module"
   },
-  "permissions": ["alarms", "tabs", "storage", "idle"],
+  "permissions": ["idle", "storage", "alarms", "tabs"],
+  "optional_host_permissions": ["https://*/*", "http://*/*"],
   "content_scripts": [
     {
-      "matches": ["http://*/*", "https://*/*", "<all_urls>"],
```

Comment:
[This article explains why we got the error](https://developer.chrome.com/docs/extensions/develop/concepts/declare-permissions#host-permissions)

Reply:
Unfortunately, with this change to make the match patterns more specific, the host permissions error is still there

Reply:
I thought I updated this PR not to have the content_scripts anymore. Sorry!